skimage/future/graph/rag.py

```diff
@@ -245,7 +251,7 @@ def copy(self):
         """Copy the graph with its max node id.
 
         .. seealso:: :func:`networkx.Graph.copy`."""
-        g = super(RAG, self).copy()
+        g = deepcopy(super(RAG, self))
```

Comment:
I think this should be ```deepcopy(self)```.  You don't need to get the super-class anymore because you aren't using a method of it.